cmd/indexd/main.go

```
@@ -233,12 +249,23 @@ func main() {
 		key := wallet.KeyFromSeed(&seed, 0)
 		fmt.Println("Recovery Phrase:", phrase)
 		fmt.Println("Address", types.StandardUnlockHash(key.PublicKey()))
+	case configCmd:
+		if len(cmd.Args()) != 0 {
+			cmd.Usage()
+			return
+		}
+
+		runConfigCmd(configPath)
 	case rootCmd:
 		if len(cmd.Args()) != 0 {
 			cmd.Usage()
 			return
 		}
 
+		if cfg.AdminAPI.Password == "" {
+			setAPIPassword()
```

Better to return an error. Prompting with stdin has always been odd, imo.